### PR TITLE
Allow aiodns on Windows

### LINKED ISSUE #8121: aiodns can probably be enabled on windows, now that it ships with c-ares already built
### Is your feature request related to a problem?

The `aiodns` library from the `[speedup] `bundle is currently disabled on Windows per [this line ](https://github.com/aio-libs/aiohttp/blob/master/setup.cfg#L64).

However, `aiodns` currently ships with `pycares`, which bundles a binary of `c-ares` by default. I was just now able to install it on Win10 with pip.

### Describe the solution you'd like

It seems there's no need to exclude Windows from the aiodns speedup. Looks like changing https://github.com/aio-libs/aiohttp/blob/master/setup.cfg#L64 would be sufficient, but I'm by no means experienced, so I'd rather not mess with the repo myself.

### Describe alternatives you've considered

There seems to be some requests to get rid of aiodns entirely in favour of pydns, but maybe till then we can use it like this?

### Related component

Client

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

### PR DESCRIPTION
**Pull request by Dreamsorcerer**
Fixes #8121.

### CODE REVIEW

**Comment by bdraco** [comment]
Protractor does not work with aiodns Last time I checked

**Comment by bdraco** [comment]
We probably need a guard to verify they’re using a selector event loop

**Comment by Dreamsorcerer** [comment]
Proactor is default right? So, test should fail..

**Comment by webknjaz** [comment]
@Dreamsorcerer @bdraco `aiodns` is now under our umbrella FYI: https://github.com/aio-libs/aiodns

File: CHANGES/9027.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Include aiodns in `[speedups]` for Windows -- by :user:`Dreamsorcerer`.
```
Comment (webknjaz) [inline CHANGES/9027.misc.rst:1]: @Dreamsorcerer FYI RST uses double backticks for this

**Pull request closed without merging**